contacts/tokens.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils import six
+
+
+class TokenGenerator(PasswordResetTokenGenerator):
```

Comment:
Why is this inherited from `PasswordResetTokenGenerator`?

Reply:
message = render_to_string('contacts/acc_active_email.html', {
                'user': user,
                'domain': current_site.domain,
                'uid': urlsafe_base64_encode(force_bytes(user.pk)).decode(),
                **'token': account_activation_token.make_token(user),**
            })

because i am using function of that here in bold line

Reply:
_make_hash_value this is the function of PasswordResetTokenGenerator which i used to create token 
`    def _make_hash_value(self, user, timestamp):
        return (
            six.text_type(user.pk) + six.text_type(timestamp) +
            six.text_type(user.is_active)
        )`